docs/application/native/guides/app-management/app-controls.md

```diff
@@ -63,18 +69,18 @@ You can launch an application with an application control using 2 different laun
 <a name="request"></a>
 ## Launch Requests
 
-To launch an application with the application control, you must create a launch request. You must create an `app_control` handle, and add to the handle the conditions for selecting the application to be launched. You can add the following information:
+To launch an application with the application control, you should create a launch request. To launch application with selected conditions you must create and add the app_control handle. You can add the following information:
 
 - Operation: Action to be performed by the launched application.
 
+  The operation is mandatory information for sending the launch request.
+
   > **Note**
   >
   > The operation name format is `http://tizen.org/appcontrol/operation/<verb>`. You can also use the related macro name, `APP_CONTROL_OPERATION_<VERB>`.
   >
   > The macro name can only be used in the `.c` files, not in the [application manifest file for the application control export](#export_appcontrol).
 
-  The operation is mandatory information for sending the launch request.
-
 - URI: Data itself to be performed.
```

Comment:
> > Line 82:
> > Definition of URI does not seems to be accurate, as I checked the information here: https://searchmicroservices.techtarget.com/definition/URI-Uniform-Resource-Identifier. Can you please update it with the accurate definition.
> 
> => Generally, URI means Uniform Resource Identifier. I understand that the author wrote in the meaning of the data of app control.

Ok, got it. If possible, can you please rephrase it to a little more descriptive way. As still it feels like more vague knowledge regarding URI.